test/functional/api/v2/smart_class_parameters_controller_test.rb

```diff
@@ -132,6 +132,43 @@ class Api::V2::SmartClassParametersControllerTest < ActionController::TestCase
     refute_equal orig_value, new_value
   end
 
+  def test_foo #test "should not update default value unless override is true" do
```

Comment:
hey! leaving silly stuff like that is my thing!
@orrabin you thing thief, you! :stuck_out_tongue_closed_eyes: